<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="navigationBG" native="true">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>-30</y>
      <width>921</width>
      <height>751</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(103, 160, 213);



</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>80</y>
       <width>231</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 50pt &quot;Bebas Neue&quot;;

</string>
     </property>
     <property name="text">
      <string>LOG IN</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="admin_user">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>220</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0,0,0,0);
border: none;
border-bottom:2px solid rgb(0, 0, 0);
color:rgba(255,255,255,230);
padding-bottom:7px;</string>
     </property>
     <property name="placeholderText">
      <string>username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="admin_pass">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>320</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0,0,0,0);
border: none;
border-bottom:2px solid rgb(0, 0, 0);
color:rgba(255,255,255,230);
padding-bottom:7px;</string>
     </property>
     <property name="placeholderText">
      <string>password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="enter">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>440</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: white;
text-size: 10px;
border-radius: 10px;
}

QPushButton:hover{
	color:rgb(73, 147, 221);
	background-color: rgb(13, 27, 38);
}</string>
     </property>
     <property name="text">
      <string>Log in</string>
     </property>
    </widget>
    <widget class="QPushButton" name="signUpButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>480</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: white;
text-size: 10px;
border-radius: 10px;
}

QPushButton:hover{
	color:rgb(73, 147, 221);
	background-color: rgb(13, 27, 38);
}
#signUpButton {
background-color: TRANSPARENT;
}
#signUpButton:hover{
color: rgb(0, 0, 255);
}</string>
     </property>
     <property name="text">
      <string>or create an account</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
